### DISCUSSION TITLE
Welcome to wifite2-requirements Discussions!

### DISCUSSION THREAD

**Discussion by KatzeMau** [KatzeMau/wifite2-requirements#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sirius-RJ** [comment]
Good God Bless you!

**Reply by jffjnfsng5464** [reply]
تهكير واي فاي

**Reply by jffjnfsng5464** [reply]
تهكير واي فاي

**Reply by jffjnfsng5464** [reply]
تهكير واي فاي
